src/site/_collections/event-schedule.js

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
You may want to move this file up to the _data directory. Alternatively, since stuff in _data is in the global namespace, it might make sense to make these [directory data files](https://www.11ty.dev/docs/data-template-dir/) that live in the `/live/` directory.

Since they're not using the eleventy `collections` object, it feels a little off to put them in the `collections` directory.

Reply:
Thanks for the suggestion. This was a bit ambitious and I'll move it to "_data" on Monday.